### PR TITLE
GPII-2161: Add`yarn.lock` file to repo...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [GPII-2161](https://issues.gpii.net/browse/GPII-2161) for details.

### CODE REVIEW

**Comment by colinbdclark** [comment]
Here's the thing, right? This might just be because I haven't finished my first coffee of the day, but I don't actually see a <code>yarn.lock</code> file in this commit. 

When I run <code>yarn install</code> on this branch, I get the following message:

    yarn install v0.17.8
    info No lockfile found.

**Comment by the-t-in-rtf** [comment]
Ugh.  I made it possible to add a yarn.lock, but did not do so.  It's fixed now.  I also went through the outdated dependencies while I was there (just a minor bump for kettle).

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "grunt": "1.0.1",
     "grunt-jsonlint": "1.1.0",
     "istanbul": "0.4.5",
-    "kettle": "1.2.1",
+    "kettle": "1.2.2",
```
Comment (colinbdclark) [inline package.json]: Snuck this one in, eh? ;)

**Comment by colinbdclark** [comment]
I ran <code>yarn install</code> and it was blazingly fast and no complaints now about the missing <code>yarn.lock</code>. And all the many tests passed beautifully when I ran <code>yarn test</code>.

**Comment by the-t-in-rtf** [comment]
Thanks for the quick help.  I'll cut the 1.0.1 release with this so I can push the new version through to other work!

**Pull request merged**